### PR TITLE
Add the statsd_count helper.

### PR DESCRIPTION
**Pull request by Natim**
Adding the patch from @greeshmab received by email.

Needs https://github.com/mozilla-services/cliquet/pull/574

### CODE REVIEW

File: syncto/views/errors.py [hunk 1]
```diff
@@ -18,11 +19,8 @@ def response_error(context, request):
                              context.response.reason,
                              context.response.text)
 
-    statsd = request.registry.statsd
-
-    if statsd:
-        statsd.count("syncclient.status_code.%s" %
-                     context.response.status_code)
+    statsd_count(request, "syncclient.status_code.%s" %
+                 context.response.status_code)
```
Comment (leplatrem) [inline syncto/views/errors.py:24]: When I look at the code just below, it makes me wonder: do we want to log a `syncclient.status_code` when response is 304 ? Shouldn't we make sure the original error comes from Sync precisely ?

**Comment by leplatrem** [comment]
Here I really see the advantage of this helper :D

**Comment by leplatrem** [comment]
r+wc

File: syncto/views/errors.py (hunk 1 above)
Comment (Natim) [inline syncto/views/errors.py:24]: > When I look at the code just below, it makes me wonder: do we want to log a syncclient.status_code when response is 304?

The idea was to log every status code (wrong or right) just have some metrics. (ie if we see that we have 70% of 304, maybe we can decide to lower the synchronization frequency or decide to prioritize the push notification before sync)

Comment (Natim) [inline syncto/views/errors.py:24]: > Shouldn't we make sure the original error comes from Sync precisely?

This is a good point, also in Syncto we are using requests only to talk with the Sync infrastructure.
I would let it like that for now, plus it is not really related to that issue.  But I will add a XXX to make sure we do not forget about it.

**Comment by Natim** [comment]
Thank you @greeshmab

**Pull request merged**